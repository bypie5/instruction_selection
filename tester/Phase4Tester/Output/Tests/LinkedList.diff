--- SelfTestCases/LinkedList.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/LinkedList.out	2020-03-14 21:20:45.000000000 -0700
@@ -14,12 +14,15 @@
 39
 25
 2220000
--555
--555
+555
+555
 28
 22
 25
 33300000
+555
+555
+28
 22
 25
 44440000
