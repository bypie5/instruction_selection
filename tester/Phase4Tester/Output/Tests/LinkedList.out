Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 53 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 55 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 57 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 85 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 87 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 89 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 111 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 133 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 149 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 185 column 14: "$t1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 190 column 14: "$t2": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 191 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 216 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 218 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 220 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 227 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 257 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 264 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 266 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 268 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 270 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 272 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 274 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 276 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 278 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 284 column 10: "0": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 290 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 292 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 323 column 10: "0": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 331 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 335 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 350 column 10: "0": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 357 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 406 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 408 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 410 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 412 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 421 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 497 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 499 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 506 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 555 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 557 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 559 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 561 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 629 column 10: "10000000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 667 column 10: "10000000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 745 column 10: "10000000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 782 column 10: "2220000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 802 column 10: "33300000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 822 column 10: "44440000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 2 column 1: Symbol "vmt_Element" not found in symbol table.
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 9 column 1: Symbol "vmt_List" not found in symbol table.
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 20 column 1: Symbol "vmt_LL" not found in symbol table.

Processing terminated due to errors.
